src/main/app/forms/validation/formValidator.ts

```diff
@@ -26,7 +27,7 @@ export class FormValidator {
     }
 
     return (req: express.Request, res: express.Response, next: express.NextFunction) => {
-      const model: T = modelTypeMapper(req.body)
+      const model: T = modelTypeMapper(removeIllegalCharacters(req.body))
```

Comment:
does everything go through the formValidator?

Reply:
I can't say for sure, but do you think it would make sense to assume that everything, that is a form submission, should?

Reply:
seems safe to me